src/core/routes/onReport.ts

```diff
@@ -1,6 +1,6 @@
 
 import { FastifyReply, FastifyRequest, FastifySchema, RouteGenericInterface } from "fastify";
-import { MessageObjectArray } from "../types/messageObjectArray";
+import { MessageObjectArray } from "../types/shared";
```

Comment:
Remove the `Types` directory as it's now useless - replace it with the shared file & rename the shared file to `Types`

Reply:
Let it remain in the types directory; it's not clear whether it is a type or not within code (because it's just named "shared").

Reply:
Ah, nevermind -- I misunderstood. Completed in https://github.com/4x8Matrix/OutSi-TS-Plugin/pull/1/commits/38b6b8f9db45b8ca9c4774339038aee60a72a11b.